lib/deploy.js

```diff
@@ -434,6 +434,49 @@ function deployScheduleTrigger(name, namespace, schedule, options) {
     });
 }
 
+function deployHttpTrigger(name, namespace, path, options) {
+  if (options.ingress.enableTlsAcme && options.ingress.tlsSecretName) {
+    throw new Error('Cannot specify both enableTlsAcme and tlsSecretName');
+  }
+  // fix for issue https://github.com/kubeless/kubeless/issues/1124
```

Comment:
This is the workaround for the issue https://github.com/kubeless/kubeless/issues/1124